ruby-lsp.gemspec

```
@@ -21,5 +21,6 @@ Gem::Specification.new do |s|
 
   s.add_dependency("language_server-protocol")
   s.add_dependency("rubocop", ">= 1.0")
+  s.add_dependency("sorbet-static-and-runtime")
```

I am not a fan of adding `sorbet-static-and-runtime` as a dependency since the gem does not really have a runtime dependency on sorbet static at all.

I'd rather that we declare this as:
```suggestion
  s.add_dependency("sorbet-runtime")
```
and add a dependency on `sorbet-static-and-runtime` in the `Gemfile` which would do the right dependabot thing that we want while still limiting our runtime dependency footprint.

---

Fair point. I've changed the dependency and added missing requires for `sorbet-runtime`.